### PR TITLE
Stops silicons from modifying circuits.

### LINKED ISSUE #38860: pAIs can interact with integrated circuits
[Round ID]: # 90356 (BAGIL)

[Reproduction]: # Be pAI, examine adjacent circuit, remove, rename or otherwise mess with it.

5a0d32c8c036968c2354f3b5ebfbc87e

[For Admins]: # (Oddities induced by var-edits and other admin tools are not necessarily bugs. Verify that your issues occur under regular circumstances before reporting them.)

### PR DESCRIPTION
**Pull request by MacHac**
Closes #38860.

:cl: MacHac
fix: Silicons, pAIs, and simple animals can no longer modify circuits.
/:cl:

Also fixes a misspelling of the word 'dexterity' that was bugging me.

### CODE REVIEW

**Comment by Time-Green** [comment]
your title fucked up

**Comment by MacHac** [comment]
whups

File: code/modules/integrated_electronics/core/assemblies.dm [hunk 1]
```diff
@@ -73,7 +73,7 @@
 		to_chat(user, "You can <a href='?src=[REF(src)];ghostscan=1'>scan</a> this circuit.");
 
 /obj/item/electronic_assembly/proc/check_interactivity(mob/user)
-	return user.canUseTopic(src, BE_CLOSE)
+	return istype(user, /mob/living/carbon) && user.canUseTopic(src, BE_CLOSE)
```
Comment (Cyberboss) [inline code/modules/integrated_electronics/core/assemblies.dm]: iscarbon

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm confused, how does this fix the bug?

**Comment by Cyberboss** [comment]
You're PR is getting old. Either Travis CI is failing, it is still labelled work in progress (change this by adding `[READY]` to the start of your title), or there are unaddressed change requests. This may be closed soon to free up space on our tracker if it remains idle. In the event that it is, please ask in #coderbus on Rizon for it to be reopened once you have resumed work on it.

**Pull request closed without merging**